patches/src/main/kotlin/app/revanced/patches/youtube/video/playback/VideoPlaybackPatch.kt

```diff
@@ -315,7 +315,7 @@ val videoPlaybackPatch = bytecodePatch(
 
         // region patch for disable VP9 codec
 
-        vp9CapabilityFingerprint.methodOrThrow().apply {
+        vp9CapabilityFingerprint.methodOrThrow(hdrCapabilityFingerprint).apply {
```

Comment:
This is an unnecessary change

Reply:
vp9CapabilityFingerprint and hdrCapabilityFingerprint are in the same class, so do it will speed up the resolve time